tools/clang/unittests/SPIRV/CodeGenSpirvTest.cpp

```diff
@@ -2385,23 +2366,9 @@ TEST_F(FileTest, VulkanLayoutTextureBufferStd430) {
   setGlLayout();
   runFileTest("vk.layout.texture-buffer.std430.hlsl");
 }
-TEST_F(FileTest, VulkanLayout64BitTypesStd430) {
-  setGlLayout();
```

Comment:
When are these tests run now that they're LIT tests? Will we be able to preserve things like this call to `setGlLayout`?

Reply:
These LIT test will run when ninja check-all or cmake --build %HLSL_BLD_DIR% --target check-all.
The setGlLayout should be translated into -fvk-use-gl-layout in the RUN line.